.github/workflows/reusable_terraform_server.yml

```diff
@@ -111,22 +115,15 @@ jobs:
         working-directory: ${{ env.TG_SRC_PATH }}/${{ inputs.environment_name }}
         run: |
           # calculate the db_cluster_snapshot_identifier, attempting to ensure it
-          # is less than 63 characters.
-          db_cluster_snapshot_identifier="pre-flyway-${{ github.ref_name }}-${{ github.sha }}"
-          if ((echo ${{ github.ref_name }} | wc -c ) > 12)
-          then
-              commit=${{ github.ref_name }}
-              db_cluster_snapshot_identifier="pre-flyway-${{ github.ref_name }}-${commit:0:7}"
-          fi
-          # remove non alphanumeric characters
+          # is less than 63 characters and has no non alphanumeric characters
+          refname=${{ github.ref_name }}
+          commit=${{ github.sha }}
+          db_cluster_snapshot_identifier="pre-flyway-${refname:0:12}-${commit:0:7}"
```

Comment:
Thanks Conrad for fixing that "commit" bug previously.
However, is there a reason to provide the upper bound "12" or should it be based on "length"?
If there is a branch name less than 12, then I wonder if it will have index-out-of-bound error (not sure in terraform)?
Thanks.

Reply:
You're right, and I didn't test that condition. I'm not really sure what that GHA syntax does. Maybe we should merge this and then open that as a small defect ticket if necessary?

Reply:
> You're right, and I didn't test that condition. I'm not really sure what that GHA syntax does. Maybe we should merge this and then open that as a small defect ticket if necessary?

That's good idea, thanks Conrad!